### PR TITLE
Substitute demos link for Simulators section

### LINKED ISSUE #687: [Documentation] Missing README file
We need a short readme file where we present on paragraph for each of the tutorials here:

https://github.com/inductiva/inductiva/tree/main/demos/tutorials

(and we should point to that readme from the main page)

"And you are good to go! You can start [exploring our tutorial notebooks](https://github.com/inductiva/inductiva/tree/main/demos)."

### PR DESCRIPTION
**Pull request by IvanPombo**
Closes #687.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ task = simulator.run(input_dir=input_dir)
 
 The user must specify the input directory containing the files to run the simulation. In the above example, a directory with the configuration of a simulation is downloaded, and passed as argument to the simulator call.
 
-Find more examples of simulations in the [tutorials section](https://github.com/inductiva/inductiva/tree/258ee549d7db93248b80632de6056fa427cff1ae/demos/tutorials).
+Find examples on how to run the others simulators [here](https://github.com/inductiva/inductiva/tree/035e20b4537a438955ce059ef8e5c90516284ed1/inductiva/simulators#simulators).
```
Comment (sarmento) [inline README.md:144]: Are you sure we need the branch info?

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ The simulators we provide are all open-source and have their own dedicated docum
 - [XBeach](https://oss.deltares.nl/web/xbeach/)
 - [GROMACS](https://www.gromacs.org/)
 
-To learn how to use these simulators with Inductiva API, check the example below and the [Simulators section](https://github.com/inductiva/inductiva/tree/main/inductiva/simulators/README.md)
+To learn how to use these simulators with Inductiva API, check the example below and the [Simulators section](https://github.com/inductiva/inductiva/tree/main/inductiva/simulators#README.md).
```
Comment (sarmento) [inline README.md]: Assuming you checked it points to the right thing, LGTM

**Review by sarmento — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (IvanPombo) [reply README.md]: Just did.

Comment (sarmento) [reply README.md]: I think you wanted this

https://github.com/inductiva/inductiva/tree/main/inductiva/simulators#simulators

Comment (IvanPombo) [reply README.md]: That is what I did, after you have already commented :)

**Pull request merged**